internal/services/netapp/_maintainers/README.md

```diff
@@ -0,0 +1,53 @@
+# This is a maintainer file for the Azure NetApp Files service.
```

Comment:
From discussing with the team, we just want this living in `netapp` and not `netapp/_maintainers`